R/pkg/DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: SparkR
 Type: Package
-Version: 3.0.0-preview
+Version: 3.0.0
```

Comment:
The PR description says like the following.
> Update the SparkR version from 3.0.0 to 3.0.0-preview

If we use `3.0.0` here, I think we cannot publish `3.0.0-preview` to `CRAN`.